### ISSUE TITLE
Notes collapse when Super+D is pressed

### ISSUE THREAD

**Issue by Bragador** [linuxmint/sticky#24]
I would like the notes not to collapse when I press Super+D. Is it possible? I'm using default Cinnamon applet "show-desktop@cinnamon.org"

**Comment by collinss** [comment]
I'm not sure if this is possible or not. Ultimately, this behavior is handled by the window manager, though there may be a way to tell the window manager that it should be ignored in that case.

**Comment by MortalHappiness** [comment]
Hi, there. I found that desklets and applets in Cinnamon will not be collapsed when super+D is pressed. Like [this desklet](https://cinnamon-spices.linuxmint.com/desklets/view/38) or [this applet](https://cinnamon-spices.linuxmint.com/applets/view/194). Is it possible to resolve this issue by wrapping sticky with applet or desklet API?

**Comment by collinss** [comment]
@MortalHappiness no, that is not possible. The technologies used are not compatible. The reason it works for desklets is that they are implemented using the same technology (and in the same process as) the window manager, so they are specifically treated differently then normal windows. It is a de-specific implementation, and will not work for this app, which should be able to work with all major desktop environments and window managers. It is completely unfeasible to apply a solution to every one of them. I've also looked into the various hints that can be passed to the window manager that might allow a cross-de solution, and there doesn't seem to be anything that would work without also causing severe and unavoidable regressions to normal use of the app.

**Comment by MortalHappiness** [comment]
@collinss Thanks. Got it. If sticky needs to be running in all major desktop environments, then it seems pretty hard to resolve this issue since there are so many window managers.
